data/tooltips.txt

```diff
@@ -840,7 +843,10 @@ tip "% shield damage / shot:"
 	`Each shot fired by this weapon does this amount of shield damage as a percentage of the target ship's total shields.`
 
 tip "% hull damage / shot:"
-	`Each shot fired by this weapon does this amount of hull damage as a percentage of the target ship's total hull.`
+	`Each shot fired by this weapon does this amount of hull damage as a percentage of the target ship or asteroid's total hull.`
```

Comment:
```suggestion
	`Each shot fired by this weapon does this amount of hull damage as a percentage of the target's total hull.`
```
Maybe? Idk.

Reply:
Feels weird to say an asteroid even has a hull.  Just because we're reusing the ship hull mechanic for asteroid health does not mean the player needs to be exposed to that.

Reply:
I'm trying to keep it consistant with the normal asteroid damage and derpy suggested mentioning damage to asteroids in the hull damage tooltip.

So I think we need a better wording rather than just leaving it or a straight revert

Reply:
I've done a re-wording myself, so I'll "resolve" this for now.